web/themes/custom/asulib_barrio/scss/components/views.scss

```diff
@@ -330,11 +346,42 @@
   }
 }
 
-.thumb-image-border .views-field-field-media-image img, .thumb-image-border .icon-container {
-  border: 1px solid #D0D0D0;
-  padding: 8px;
+.collection-browse-object .icon-container a img {
+  max-width: 200px;
 }
 
 .main-content .thumb-image-border a {
   border-bottom: none;
-} 
+}
+
+.view-content-recent .icon-container {
+  height: 100%;
+  object-fit: $uds-component-card-basic-image-top-object-fit;
+}
+
+.view-id-content_recent table td {
+  border: none;
+  padding: 0;
+}
+
+/*
+.vert-image-frame {
+  background-image: linear-gradient(to bottom, #ffffff, #f9f9f9, #f3f3f3, #ededed, #e7e7e7);
+} */
+
+.node--view-mode-recent-item-teaser-prism .vert-image-frame .field-content {
+  position: relative;
+  height: 300px;
+}
+
+.node--view-mode-recent-item-teaser-prism .vert-image-frame img {
```

Comment:
whats this section doing?

Reply:
I think it was overlooked in the hundreds of changes between the several branch -- it is also in the other branch that is pending on KEEP images (recent items), so would you say to remove the section here?

Reply:
yeah this file appears to have the same changes as https://github.com/asulibraries/islandora-repo/pull/451/files#diff-ead1f9a48d15332377148fc624038daa09cc0c2a219299119154c26eedadad3b

Reply:
good-o. I will push a new version here that no longer has it.